packages/ckeditor5-utils/src/config.ts

```diff
@@ -193,9 +193,9 @@ export default class Config<Cfg> {
 
 		// Iterate over parts to check if currently stored configuration has proper structure.
 		for ( const part of parts ) {
-			// If there is no object for specified part then create one.
-			if ( !isPlainObject( target[ part ] ) ) {
-				target[ part ] = Object.create( null );
+			// If target[part] is not an object or array, initialize as empty object.
+			if ( typeof target[ part ] !== 'object' || target[ part ] === null ) {
+				target[ part ] = {};
```

Comment:
Why such a change? This was changed recently to avoid prototype pollution.

Reply:
I agree with @niegowski comment. This part of the code is within the mitigation area for prototype pollution, which is also flagged by code scanners. The change is fully intentional. Bypassing security measures just to apply a bugfix isn’t a good practice. It’s better to fix this on the side of the library that relies on this behavior.